mkdocs/docs/HPC/setting_up_python_virtual_environments.md

```diff
@@ -0,0 +1,221 @@
+# Python Virtual Environments (venv's)
+
+## Introduction
+
+Python virtual environments (venv's) are a way to create isolated Python environments.
```

Comment:
```suggestion
A Python virtual environment (`venv` for short) is a way to create an isolated Python environment.
```

The last "environment" is a bit too enigmatic imho, especially since "environment" also has a totally different outside of Linux, try to rephrase/expand that a bit?

Reply:
Applied commit and rephrased last 'environments' in [87a30af](https://github.com/hpcugent/vsc_user_docs/pull/639/commits/87a30afc16070d39726aa37139fbdc62b1162432):

> A Python virtual environment (venv for short)
is a tool to create an isolated Python workspace with its own dependencies.